src/scripts/main.js

```diff
@@ -1 +1,14 @@
 'use strict';
+
+const largeImg = document.querySelector('#largeImg');
+const listItem = document.querySelectorAll('#thumbs a');
+
+listItem.forEach((thumb) => {
+  thumb.addEventListener('click', () => {
+    event.preventDefault(); 
+
+    const newSrc = thumb.getAttribute('href');
+
+    largeImg.src = newSrc;
+  });
+});
```

Comment:
Code Quality Issue: It's better to use event delegation in this case. Instead of assigning an event listener to each thumbnail, you can assign one event listener to their parent element. When a click event occurs, you can use `event.target` to determine which thumbnail was clicked. This approach will improve performance as it reduces the number of event listeners.